main.css

```diff
@@ -1,76 +1,427 @@
+    margin-top: 5%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+  }
+
+  .see-project-button:hover {
+    box-shadow: 0 4px 8px 0 #36b37f69;
+  }
+
+  .see-project-button:active {
+    background-color: #008552;
+  }
+
+  .personal-works {
+    display: flex;
+    flex-direction: column;
     align-items: center;
-    padding-top: 20%;
+  }
+
+  .about-me-detail {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .about-me-head {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    background-position: center;
+    font-family: "Inter", sans-serif;
+    margin-top: 192px;
+    background-image: url("images/abouthead.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 300px;
+    width: 100%;
+  }
+
+  .about-me-head h2 {
+    font-size: 30px;
+    color: #172b4d;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .about-me-head p {
+    margin-left: 5%;
+    margin-right: 5%;
+    color: #42526e;
+    margin-top: 24px;
+    width: 900px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .aboutme-button {
+    background-color: #36b37f;
+    color: white;
+    border: none;
+    border-radius: 4px;
+    padding: 12px;
+    margin-top: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+  }
+
+  .aboutme-button:hover {
+    box-shadow: 0 4px 8px 0 #36b37f69;
+  }
+
+  .aboutme-button:active {
+    background-color: #008552;
+  }
+
+  .category-container {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    gap: 24px;
+    margin-top: 52px;
+  }
+
+  .category-card {
+    border: 2px solid #d0d9d4;
+    background-color: #ebf0ee;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-top: 6%;
+  }
+
+  .skills-list-2 {
+    margin-top: 8%;
+    display: flex;
+    list-style: none;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 1%;
+  }
+
+  .skills-list-2 li {
+    background-color: white;
+    border-radius: 8px;
+    padding: 8px 12px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    color: #36b37e;
+    margin: 5px;
+  }
+
+  .single-card-2 h3 {
+    font-family: "Inter", sans-serif;
+    font-size: 24px;
+    font-weight: 600;
+    font-style: normal;
+    color: #3a4a42;
+    margin-top: 10%;
+  }
+
+  .single-card-2 {
+    width: 369px;
+    height: 352;
+  }
+
+  footer form {
+    display: grid;
+    grid-template-columns: auto auto;
+    row-gap: 24px;
+  }
+
+  footer {
+    margin-top: 164px;
+    background-image: url("images/footerbg.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  footer ul {
+    list-style: none;
+  }
```

Comment:
Expected selector "footer ul" to come before selector ".top-guide ul"   no-descending-specificity